releasenotes/notes/create-ddot-oci-678f8e0c29ccd48c.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Added support for building, packaging, and installing ddot as an OCI package. The project can now
+    be installed using the image.
```

Not sure about using "DDOT" -- is this customer-facing?

```suggestion
    Added support for building, packaging, and installing the DDOT as an OCI package. The project can now
    be installed using the Datadog Installer.
```